client/components/gsuite/gsuite-new-user-list/style.scss

```diff
@@ -1,10 +1,17 @@
-.gsuite-new-user-list__new-user-remove-user-button {
-	width: 100%;
-	@include breakpoint-deprecated( '>800px' ) {
-		width: 75px;
+.gsuite-new-user-list__new-user {
+	fieldset {
+		margin-bottom: 0;
 	}
+}
+
+.gsuite-new-user-list__new-user-remove-user-button {
+	display: none; // TODO: Fix position of button which is displayed in the middle of input fields in small viewports
```

Comment:
Is it customary to add a ticket for this TODO and include the issue in this comment as a reference?

Reply:
That's a good idea, I'll update that comment. Thanks for the review @krymson24!